account_tax_repartition_line_tax_group_account/models/account_tax_repartition_line.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 ForgeFlow, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+
+
+class AccountTaxRepartitionLine(models.Model):
+    _inherit = "account.tax.repartition.line"
+
+    @api.model
+    def create(self, vals):
+        account_id = vals.get("account_id", "")
+        repartition_type = vals.get("repartition_type", "")
+        tax_id = vals.get("tax_id", "")
+        if not account_id and tax_id and repartition_type == "tax":
+            tax = self.env["account.tax"].browse(tax_id)
+            def_acc = tax.tax_group_id.with_context(
+                force_company=tax.company_id.id
```

Comment:
```suggestion
                with_company=tax.company_id.id
```
Maybe change it to 'with_company' since 'force_company' is no longer supported in this version

Reply:
updated, thanks